kolbot/libs/bots/Mausoleum.js

```diff
@@ -23,16 +23,16 @@ function Mausoleum() {
 		throw new Error("Failed to move to Mausoleum");
 	}
 
-	Attack.clearLevel(Config.ClearType);
+	//Attack.clearLevel(Config.ClearType);
 
-	if (Config.Mausoleum.ClearCrypt) {
+	/*if (Config.Mausoleum.ClearCrypt) {
```

Comment:
Maybe adding a Config option and a conditional check here instead of commenting out the main concept of this script would be more ideal to cater to everyones needs. (Maybe some _do_ want to clear Mausoleum?)